roles/pulp_repos/defaults/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+
+# defaults file for pulp_repos
+pulp_rhel_codeready_repo:
+  - codeready-builder-for-rhel-8-x86_64-rpms
+  - rhui-codeready-builder-for-rhel-8-rhui-rpms
+  - codeready-builder-for-rhel-8-rhui-rpms
+rhel7_optional_repo:
```

Comment:
rhel7_optional_repo and epel_release_packages need their documentation moved from https://github.com/pulp/pulp_installer/tree/main/roles/pulp_common#role-variables